ghedesigner/search_routines.py

```diff
@@ -206,8 +206,8 @@ def search(self):
         if check_bracket(sign(t_0_lower), sign(t_0_upper)):
             if self.disp:
                 print("Size between min and max of lower bound in domain.")
-            self.initialize_ghe(self.coordinates_domain[0], self.sim_params.max_height)
-            return 0, self.coordinates_domain[0]
+            self.initialize_ghe(self.coordinates_domain[x_l_idx], self.sim_params.max_height)
+            return x_l_idx, self.coordinates_domain[x_l_idx]
```

Comment:
Another overlap of #99 and #100 that didn't work together. Duplicated indices tracking the upper and lower bounds were causing the new `max_boreholes` and `continue_if_design_unmet` input fields to not work together. This fixes it.